nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -738,94 +738,86 @@ phNciNfc_ResetNtfCb(void*     pContext,
         PH_LOG_NCI_INFO_STR("Received RESET notification from NFCC");
 
         if (pNciCtx->dwNtfTimerId != 0)
-        {
             (void)phOsalNfc_Timer_Stop(pNciCtx->dwNtfTimerId);
 
-            if (pTransInfo->wLength >= PHNFCINFC_CORE_RESET_NTF_MIN_LEN)
+        /* Reset Sender statemachine */
+        (void)phNciNfc_CoreResetSenderStateMachine(&pNciCtx->NciCoreContext);
+        (void)phTmlNfc_WriteAbort(pNciCtx->NciCoreContext.pHwRef);
+
+        if (NULL != pNciCtx->tRegListInfo.pResetNtfCb)
+        {
+            pNciCtx->tRegListInfo.pResetNtfCb(pNciCtx->tRegListInfo.ResetNtfCtxt,
+                eNciNfc_NciResetNtf, pInfo, NFCSTATUS_SUCCESS);
+        }
+
+        if (pTransInfo->wLength >= PHNFCINFC_CORE_RESET_NTF_MIN_LEN_2x)
```

Comment:
In the case of NCI1x pTransInfo->wLenght of the CORE_RESET_NTF is < PHNCINFC_CORE_RESET_NTF_MIN_LEN_2x and the Nci version is not available is that frame.

If ok for you I will add a comment to clarify the condition.
I have spotted a spelling issue in PHNFCINFC_CORE_RESET_NTF_MIN_LEN_2x, i will fix that as well.

Also I will add bracket around phOsalNfc_Timer_Stop.